app/forms/qualification_details_form.rb

```diff
@@ -0,0 +1,143 @@
+class QualificationDetailsForm < Form
+  attribute :qualifications_details_check, :boolean
+
+  validates :qualifications_details_check,
+    inclusion: {
+      in: [true, false],
+      message: ->(form, _) { form.i18n_errors_path("qualifications_details_check") }
+    }
+
+  def initialize(claim:, journey:, params:)
+    super
+
+    self.qualifications_details_check = permitted_params.fetch(
+      :qualifications_details_check,
+      claim.qualifications_details_check
+    )
+  end
+
+  def save
+    return false unless valid?
+
+    claim.assign_attributes(
+      qualifications_details_check: qualifications_details_check
+    )
+
+    if qualifications_details_check
+      # Teacher has confirmed the details in the dqt record are correct, update
+      # the eligibility with these details
+      claim.claims.each { |c| set_qualifications_from_dqt_record(c.eligibility) }
+    else
+      # Teacher has said the details don't match what they expected so
+      # nullify them
+      claim.claims.each { |c| set_nil_qualifications(c.eligibility) }
+    end
+
+    claim.save!
+  end
+
+  def dqt_route_into_teaching
+    claim.dqt_teacher_record.route_into_teaching
+  end
+
+  def dqt_academic_date
+    AcademicYear.for(claim.dqt_teacher_record.academic_date)
+  end
+
+  # Often the DQT record will represent subject names in all lowercase
+  def dqt_itt_subjects
+    claim.dqt_teacher_record.itt_subjects.map do |subject|
+      (subject.downcase == subject) ? subject.titleize : subject
+    end.join(", ")
+  end
```

Comment:
I don't understand why we cannot `titlteize` all of them 🤔

Reply:
I'm not sure of the business rules maybe some are all in CAPS and we want to keep that format?

Reply:
Never seen that :D Safe to keep in case, I originally thought it was for some optimisation reason but if anything it probably makes it even worse. Not a blocker